src/ForkCMS/Utility/Thumbnails.php

```diff
@@ -157,4 +147,87 @@ public function getFolders(string $inPath, bool $includeSourceFolder = false): a
 
         return array_values($folders);
     }
+
+    /**
+     * Calculate the desired width based on the aspect ratio if needed
```

Comment:
Isn't this useless phpdoc? The method signature explains it very well I would think

Reply:
removed it in 14c6ad269